src/content/docs/how-to/local.mdx

```diff
@@ -1,12 +1,171 @@
-This page is a draft, so contents are unfinished or nonexistent.
+This section is a draft, so contents are unfinished or nonexistent. The CLI interface may be refactored in the future; see [here][refactor-discussion] for more information.
 :::
 
-https://diataxis.fr/how-to-guides/
 
--   `bb build --rebase` (or whatever the correct command is)
+### Build
+
+The `build` command builds an image from a recipe; you can use this command to test your recipe locally before pushing the recipe to GitHub:
+
+```bash
+bluebuild build ./config/recipe.yml
+```
+
+You can see that the BlueBuild GitHub Action [uses the same `build` command][bluebuild-cli-ghaction-build], with additional `--push` and `--registry` options for pushing successful build to `ghcr.io` or similar image repositories.
+
+You can see the complete list of available options using `bluebuild build help`:
+
+```bash
+bluebuild build [OPTIONS] [RECIPE]
+
+Build an image from a recipe
+
+Usage: bluebuild build [OPTIONS] [RECIPE]
+
+Arguments:
+  [RECIPE]  The recipe file to build an image
+
+Options:
+  -p, --push
+          Push the image with all the tags
+  -r, --retry-push
+          Allow `bluebuild` to retry pushing images if it fails
+      --retry-count <RETRY_COUNT>
+          The number of times to retry pushing the image [default: 1]
+  -f, --force
+          Allow `bluebuild` to overwrite an existing Containerfile without confirmation
+  -a, --archive <ARCHIVE>
+          Archives the built image into a tarfile in the specified directory
+      --registry <REGISTRY>
+          The registry domain name
+      --registry-namespace <REGISTRY_NAMESPACE>
+          The url path to your base project images [aliases: registry-path]
+  -v, --verbose...
+          Increase logging verbosity
+  -q, --quiet...
+          Decrease logging verbosity
+  -U, --username <USERNAME>
+          The username to login to the container registry
+  -P, --password <PASSWORD>
+          The password to login to the container registry
+  -c, --connection <CONNECTION>
+          The connection string used to connect to a remote podman socket
+  -h, --help
+          Print help (see more with '--help')
+```
+
+### Upgrade
+
+:::caution
+This section is a draft, so contents are unfinished or nonexistent. The CLI interface may be refactored in the future; see [here][refactor-discussion] for more information.
+:::
+
+
+### Rebase
+
+:::caution
+This section is a draft, so contents are unfinished or nonexistent. The CLI interface may be refactored in the future; see [here][refactor-discussion] for more information.
+:::
+
+
+### Bug-Report: Getting Help
+
+You can autogenerate a GitHub Issue with OS/environment/version metadata using the `bug-report` command:
+
+```bash
+$ bluebuild bug-report
+```
+
+The command will ask for recipe information and will present a confirmation dialog before generating a draft issue. You can see an example of an issue generated using this command [here][sample-bug-report].
```

Comment:
This is also something mostly irrelevant to this page. I've been meaning to add this to the [troubleshooting page](https://blue-build.org/learn/troubleshooting/), which I've been slowly filling in at recently.

Reply:
Fair enough, removed this in latest commit